libs/utilities/src/lib/axios-error/axios-error.ts

```diff
@@ -0,0 +1,12 @@
+import { AxiosError } from 'axios';
+
+export const errorToAxiosError = (
+  error: unknown,
+  key = 'reason',
+): string | undefined => {
```

Comment:
The return type should be `AxiosError` so we can still access the full axios object. It includes HTTP codes and other important details.

```suggestion
): AxiosError<{ [key: string]: string }> => {
```